setup.py

```diff
@@ -29,7 +29,7 @@
         'backports.shutil_get_terminal_size'
     ],
     setup_requires=['pytest-runner'],
-    tests_require=['pytest >= 2.9, < 3'],
+    tests_require=['pytest >= 2.9'],
```

Comment:
Well, things didn't appear to break between 2.9 and 3.0, so I think it's best to stick with the lowest working value that we know of.